# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../unison/tail.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ruN unison-2.9.20/Makefile.OCaml unison-2.9.20-new/Makefile.OCaml
--- unison-2.9.20/Makefile.OCaml	2002-08-19 23:15:04.000000000 +0200
+++ unison-2.9.20-new/Makefile.OCaml	2004-05-24 12:43:49.546990904 +0200
@@ -58,7 +58,7 @@
 # to Unix '/' separators, and extraneous control-M's are deleted.
 # Unfortunately there is a literal control-M buried in this, I'd rather
 # get rid of it...
-OCAMLLIBDIR=$(shell ocamlc -v | tail -1 | sed -e 's/.* //g' | sed -e 's/\\/\//g' | sed -e 's///g')
+OCAMLLIBDIR=$(shell ocamlc -v | tail -n 1 | sed -e 's/.* //g' | sed -e 's/\\/\//g' | sed -e 's///g')
 
 # User interface style: 
 #   Legal values are
